### PR TITLE
CMS api now serves HTML

### PR DESCRIPTION
**Pull request by asmega**
- At the moment the CMS generates markdown content which is then processed by the client.
- This changes the API so it is now the job of the CMS to generate HTML.
- I think this is the job of the CMS and not the client hence the move
- This will save every client from having implement something to process the markdown (with custom snippets) to HTML

### CODE REVIEW

File: spec/models/block_composer_spec.rb [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      let(:blocks) { [] }
+
+      it 'is an empty string' do
+        expect(composer.to_html).to eql("\n")
+      end
+    end
+
+    context 'non-content block provided' do
+      let(:non_content_block) do
+        { 'identifier' => 'not_content', 'content' => 'Definitely not a Content Block' }
```
Comment (tomas-stefano) [inline spec/models/block_composer_spec.rb]: Yeah. Definitely! :)

**Comment by tomas-stefano** [comment]
:+1: :+1:

File: features/step_definitions/edit_page_steps.rb [hunk 2]
```diff
@@ -9,7 +9,7 @@
 end
 
 Then(/^I should see the Draft Article$/) do
-  expect(JSON.parse(preview_page.text)['blocks'].find {|b| b["identifier"] == 'content'}['content']).to eq('test')
+  expect(JSON.parse(preview_page.text)['blocks'].find {|b| b["identifier"] == 'content'}['content']).to include('test')
```
Comment (tomas-stefano) [inline features/step_definitions/edit_page_steps.rb]: This assertion is too smart. I know that was already there. I would move this to a less smart assertion.

File: spec/models/block_composer_spec.rb (hunk 1 above)
Comment (asmega) [inline spec/models/block_composer_spec.rb]: I just borrowed this from frontend. So not my code. I'll delete from frontend once this is integrated.

File: features/step_definitions/edit_page_steps.rb (hunk 2 above)
Comment (asmega) [inline features/step_definitions/edit_page_steps.rb]: Not gonna touch this at the moment.

File: spec/models/block_composer_spec.rb (hunk 1 above)
Comment (tomas-stefano) [inline spec/models/block_composer_spec.rb]: I don't have any problem with this line. For me it's fine. It just a funny and a bad comment.

File: features/step_definitions/edit_page_steps.rb (hunk 2 above)
Comment (tomas-stefano) [inline features/step_definitions/edit_page_steps.rb]: Ok. For me it's fine this PR. :+1:

**Pull request merged**